ui/src/app/components/workspace/workspace.component.html

```diff
@@ -17,12 +17,14 @@
     </div>
   </div>
   <div class="container">
-    <h2>Configure schema</h2>
+    <h2 class="vertical-center">Configure schema</h2>
     <div class="summary">
       Estimation for {{ srcDbName.toUpperCase() }} to Spanner conversion:
       {{ conversionRatePercentages.good }}% of tables can be converted automatically,
       {{ conversionRatePercentages.ok }}% requires minimal conversion changes and
       {{ conversionRatePercentages.bad }}% requires high complexity conversion changes.
+      <br>
+      To make schema changes go to <a class="spanner-tab-link-workspace" (click)="spannerTab()">Spanner Draft pane</a>.
```

Comment:
add link in Spanner Draft instead of Spanner Draft pane

Reply:
Updated